docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -806,4 +806,163 @@ The following is performed on ArduCopter version v4.5.7
+
+Then, we need to find how much data remains which we can find by taking:
+
+16 bytes - (2 full frames * 7 bytes/frame) = 2 data bytes remaining.
+
+For this last frame, we need to account for the tail byte since we're not sending a full frame, so we'll have 3 total bytes of data in this final frame. Finding the total 
+number of bits is then done by:
+
+(131 bit/full frame * 2 full frame) + (67 bit/empty frame + (3 data bytes * 8 bit/byte)) = 353 bits
+
+This example is visualized in the DroneCAN GUI Tool's bus monitor here:
+
+.. image:: ../_static/comms_protocols_pictures/dronecan/multi_frame_raw_cmd.png
+
+Now, we can once again find our utilization of 1 second at 500kbit by:
+
+353 bit * 1/500Kbps * 100% = 0.071% Utilization
+
+We can generalize utilization of RawCommands of more than 4 unique ESCs as the following steps:
+
+* Bytes for All Commands (C) = 1.75 bytes/command * (Number of Commands) + 2 bytes CRC
+* Number of full frames (F) = floor(C / 7)
+* Total bytes in last  frame (L) = C - 7F + 1 
+* Total bits (b) = 131 bitfull frame * F + (67 bit/empty frame + 8L)
+* Utilization of 1 Second (%) = b * 1/bitrate * Command Frequency (Hz) * 100%
+
+For 6 commands:
+
+.. csv-table:: Six Raw Command Bus Utilization
+	:file: ../_static/comms_protocols_pictures/dronecan/six_raw_cmds.csv
+	:header-rows: 1
+
+For 8 commands:
+
+.. csv-table:: Eight Raw Command Bus Utilization
+	:file: ../_static/comms_protocols_pictures/dronecan/eight_raw_cmds.csv
+	:header-rows: 1
+
+DroneCAN Utilization of a Quadcopter
+======================================
+Now that we know how to calculate the utilizations of various DroneCAN messages, it is easy to extend our understanding to an entire vehicle of DroneCAN nodes. In order to do 
+so, we simply need to sum together the impacts of each message. Let's suppose that our flight controller transmits RawCommand messages at 400Hz, and that each of our modules 
+transmits telemetry at 10Hz.
+
+Since we are controlling four modules, we'll need four commands per RawCommand message, and we'll need only one frame to send them. Using our calculation :ref:`above <raw_cmd_utilization>`, 
+we find that the utilization of these commands are:
+
+.. csv-table:: Quad Example RawCommand Bus Utilization
+	:file: ../_static/comms_protocols_pictures/dronecan/quad_ex_raw_cmd.csv
+	:header-rows: 1
+
+Then, we can use our :ref:`telemetry calculation <telemetry_utilization>` in order to find the impact of the four modules reporting telemetry as:
+
+.. csv-table:: Quad Example Telemetry Bus Utilization
+	:file: ../_static/comms_protocols_pictures/dronecan/quad_ex_telem_utilization.csv
+	:header-rows: 1
+
+After summing, we find the total utilization as:
+
+.. csv-table:: Quad Example Total Bus Utilization
+	:file: ../_static/comms_protocols_pictures/dronecan/quad_ex_total_utilization.csv
+	:header-rows: 1
+
+In general, our testing suggests keeping your utilization below 80% for the best possible performance.
```

Comment:
Do we actually have any testing that recommends this? Or a source we can cite or anything? If there isn't any trustworthy thing we can dig up on it, maybe just lose it